translations/German/general-notes.md

```diff
@@ -0,0 +1,82 @@
+# General Rules
+
+Thank you for considering to contribute to the German translation!
+Before you start writing, please make sure that you have read the
+following general translation rules.
+
+## Formal or Informal?
+
+**Informal.**
+
+Although written German tends to be quite formal, websites in German are
+usually following informal netiquette. As Zulip's guides are written
+in a more colloquial style, German translations should be rather informal as well.
+
+**Don't use slang or regional phrases in the German translation:**
+* ~~"So'n Dreck kann jedem mal passieren."~~
+* ~~"Das ist die Seite, wo der Quelltext steht."~~
+
+## Form of address
+
+**Use "Du" instead of "Sie".**
+
+For the reasons provided in #Formal_or_Informal, stick to *Du*(informal)
+instead of *Sie*(formal) when addressing the reader.
+Always capitalize the *Du* when addressing him, do **not** use *du*.
+
+## Form of instruction
+
+**Prefer imperative over constructions with auxiliary verbs.**
+
+For instructions, try to use the imperative (e.g. *"Gehe auf die Seite"* -
+*"Go to the page"*) instead of constructions with auxiliary verbs
+(e.g. *"Du musst auf die Seite ... gehen"* - *"You have to go the page ..."*).
+This keeps the phrases short and avoids unnecessary addressings of the reader.
+
+## Form of titles
+
+**Use continuous titles with verbs in infinitive form**
+
+To be consistent with other online platforms, use continuous titles with verbs
+in infinitive form, e.g. *Manage streams* - *Kanle verwalten* instead of
+~~*Verwalte Kanle*~~.
+
+##Concatenation of words
+
+**Try to avoid them.**
+
+The German language is infamous for its concatenations of nouns
+(e.g. "Heizölrückstoßdämpfung", which means "fuel oil recoil attenuation").
+For the sake of correct rendering and simplicity, you should try to avoid such
+concatenations whenever possible.
+
+## Anglicisms
+
+**Use them if a millenial would understand them better than the German equivalent.**
```

Comment:
I'm not sure what your definition of a `millenial` is, but I've heard recently it's those born in the 80's - I don't think you meant that here. A `teenager` might be slightly better, though still a bit baffling for older translators - teenagers tend to use more anglicisms and slang words.

Also, not sure what "understand better" means - e.g. I'd understand "Kanal" and "Stream" just the same, but I have a slight preference for the English word. I know it's hard to define perceived awkwardness. I'd go for "is more widely used in web apps".

This general guide would benefit from the `Task Type A`, which is research for translations - are you planning to take on that task?